docs/pages/access-controls/idps/saml.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Using Teleport as a SAML identity provider
+description: How to configure and use Teleport as a SAML identity provider.
+h1: Teleport as a SAML identity provider
+---
+
+## How to use Teleport as an identity provider
+
+This guide will cover how to use Teleport as an identity provider so that Teleport
+can be used to authenticate to external services. 
+
+## Prerequisites
+
+- Teleport role with access to maintaining `saml_idp_service_provider` resources.  This is available in the default `editor` role.
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Configuring the SAML identity provider service
+
+The SAML identity provider service is enabled by default on the proxy service
```

Comment:
Is there a way to disable it if you don't want this functionality?

Reply:
There is. Talking with @alexfornuto, this is better placed in the config reference, which I can add.